src/app/Components/PrivateRoute/index.js

```diff
@@ -1,18 +1,15 @@
 import React from 'react';
 import { Route, Redirect } from 'react-router-dom';
 import { isLoggedIn } from '../../lib/Cookie';
+import { setFeatureFlag } from '../../lib/FeatureFlag';
 
-const PrivateRoute = ({ component: Component, ...rest }) => (
-  <Route
-    {...rest}
-    render={props =>
-      isLoggedIn() === true ? (
-        <Component {...props} />
-      ) : (
-        <Redirect to="/login" />
-      )
-    }
-  />
-);
+const PrivateRoute = ({ component: Component, ...rest }) => {
+  if (isLoggedIn() === true) {
+    if (window.location.hash) setFeatureFlag(window.location.hash.slice(1));
+    return <Route {...rest} render={props => <Component {...props} />} />;
+  } else {
+    return <Route {...rest} render={() => <Redirect to="/login" />} />;
+  }
```

Comment:
Not sure how you feel about catch-all `else` clauses but this could be written without the else.

Reply:
> I'm guessing the babel config is needed to get Jest to play nicely?

yeah man